diff -ur happy-1.10.orig/aclocal.m4 happy-1.10/aclocal.m4
--- happy-1.10.orig/aclocal.m4	Mon Apr 23 15:35:18 2001
+++ happy-1.10/aclocal.m4	Tue Sep 18 20:22:46 2001
@@ -638,7 +638,7 @@
 
 dnl *** Helper function **
 dnl 
-AC_DEFUN(FPTOOLS_IN_SCOPE,AC_TRY_LINK([extern char* $1;],[return (int)&$2], $3=yes, $3=no))
+AC_DEFUN([FPTOOLS_IN_SCOPE],[AC_TRY_LINK([extern char* $1;],[return (int)&$2], $3=yes, $3=no)])
 
 dnl *** What's the end-of-text-section marker called? ***
 dnl
@@ -767,25 +767,6 @@
 []
 )])]
 )
-
-dnl check for prototypes
-dnl
-AC_DEFUN(AC_C_PROTOTYPES,
-[AC_CACHE_CHECK([prototypes], ac_cv_prototypes,
-[AC_TRY_COMPILE([
-void foo(int);
-void foo(i)
-int i; { 
-return;
-}
-],
-[int i;], 
-ac_cv_prototypes=yes,
-ac_cv_prototypes=no)])
-if test "$ac_cv_prototypes" = yes; then
-AC_DEFINE(HAVE_PROTOTYPES)
-fi
-])
 
 dnl ** Check which CATALOG file we have to use with DocBook SGML.
 dnl
