diff -urN happy-1.14.org/configure.ac happy-1.14/configure.ac
--- happy-1.14.org/configure.ac	2004-06-13 10:08:05.050570904 +0200
+++ happy-1.14/configure.ac	2004-06-13 10:12:12.153005624 +0200
@@ -87,7 +87,7 @@
         HostVendor_CPP='dec'
         HostOS_CPP='osf3'
         ;;
-alpha*-unknown-linux*)
+alpha*-*-linux*)
 	HostPlatform=alpha-unknown-linux
 	TargetPlatform=alpha-unknown-linux
 	BuildPlatform=alpha-unknown-linux
@@ -341,7 +341,7 @@
 	HostVendor_CPP='apple'
 	HostOS_CPP='darwin'
 	;;
-powerpc-unknown-linux*)
+powerpc-*-linux*)
         HostPlatform=powerpc-unknown-linux
         TargetPlatform=powerpc-unknown-linux
         BuildPlatform=powerpc-unknown-linux
